internal/manifests/collector/adapters/config_to_ports.go

```diff
@@ -93,6 +94,8 @@ func ConfigToComponentPorts(logger logr.Logger, cType ComponentType, config map[
 			cmptParser, err = exporterParser.For(logger, cmptName, exporter)
 		case ComponentTypeReceiver:
 			cmptParser, err = receiverParser.For(logger, cmptName, exporter)
+		case ComponentTypeProcessor:
+			logger.V(2).Info("processors don't provide a way to enable associated ports", "name", key)
```

Comment:
I think this should be a debug log (`V(4)`), otherwise this is going to get spammy